app/domain/authentication/authn_jwt/signing_key/signing_key_settings.rb

```diff
@@ -0,0 +1,23 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      class SigningKeySettings
+        def initialize(signing_key_uri:,
+                       signing_key_type:)
+          @signing_key_uri = signing_key_uri
+          @signing_key_type = signing_key_type
+        end
+
+        def signing_key_uri
+          @signing_key_uri
+        end
+
+        def signing_key_type
+          @signing_key_type
+        end
```

Comment:
Please replace custom getters with `attr_reader`. See the [article](https://www.rubyguides.com/2018/11/attr_accessor/).

Reply:
May be it will be better to call them just `uri` and `type`.
`SigningKeySettings.signing_key_uri` vs `SigningKeySettings.uri`
`SigningKeySettings.signing_key_type` vs `SigningKeySettings.type`

WDYT?

Reply:
how about also changing the class name to `SigningKey`? i think that the `Settings` is redundant.

Reply:
I'm actually doubting. The class does not represent a signing key itself, only the properties those help to fetch a signing key. So naming the class `SigningKey` can be confusing.
The class in the next stage will have `ca_cert` property for example.
@orenbm do you have better name for this purpose?